mssql/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/mssql/server:2017-latest
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY . /usr/src/app
```

Comment:
COPY and RUN should be the last two commands in the Dockerfile since they change.